spring-cloud-gcp-samples/spring-cloud-gcp-config-sample/pom.xml

```diff
@@ -2,16 +2,31 @@
 <project xmlns="https://maven.apache.org/POM/4.0.0"
          xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="https://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.2.0.BUILD-SNAPSHOT</version>
```

Comment:
Maybe use a release version of Spring Boot instead?

Reply:
Looks like the version used in the project currently is `2.2.0.M4`. So should we use that instead? Or is snapshot good for helping them test their project?